plugins/normalize-whitespace/index.html

```diff
@@ -103,6 +103,16 @@ <h1>Examples</h1>
 		};
 
 
+		var
```

Comment:
Why did you add this here? This seems unrelated to your changes.

Reply:
Are you asking about the `var` or the entire thing? this `var` is in initilzizing the `there_is_a_very_very_very_very_long_line_it_can_break_it_for_you` variable. The original without line breaks is in  `demo.html` line 25

The reason this block is here is it is demoing using `data-break-lines`